supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -173,6 +196,12 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
+  private def generateGcsEgressCheckScript(inputs: List[PipelinesApiInput])
+                                          (implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
+    val bundleFunction = (gcsEgressCheckBundle(gcsTransferConfiguration) _).tupled
+    generateGcsTransferScript(inputs, bundleFunction) |> bracketTransfersWithMessages("Egress Check")
```

Comment:
`generateGcsTransferScript` should probably be renamed if it's going to be used for non-transfer purposes...

Reply:
I've added the egress check bash function to the transfer bash script (which is used as a library of bash functions). The egress check is a "pre-flight" check for transfer, so it feels suitable to include in the transfer script. This is why I needed to include all the transfer library code.